def.bzl

```diff
@@ -17,42 +17,43 @@ load("//internal:go_repository.bzl", "go_repository")
 load("//internal:overlay_repository.bzl", "git_repository", "http_archive")
 
 def _gazelle_runner_impl(ctx):
-  args = [
-      ctx.attr.command,
-      "-mode", ctx.attr.mode,
-      "-external", ctx.attr.external,
-  ]
-  if ctx.attr.prefix:
-    args.extend(["-go_prefix", ctx.attr.prefix])
-  if ctx.attr.build_tags:
-    args.extend(["-build_tags", ",".join(ctx.attr.build_tags)])
-  args.extend(ctx.attr.extra_args)
+    args = [
```

Comment:
In general, I'd prefer formatting changes to be in separate PRs. It makes other changes easier to rebase and cherry-pick. I don't think this particular change will cause any harm though, so need to split it this time.